optados/test-suite/tools/parsers/parse_dos_dat.py

```diff
@@ -0,0 +1,53 @@
+"""
+Parser function parse() to parse the .wout output file of Wannier90.
+"""
```

Comment:
might want to change this line - and similar in the others parsers

Reply:
Done! in updated PR